dubbo-samples-rest/src/main/java/org/apache/dubbo/integration/swagger/DubboSwaggerApiListingResource.java

```diff
@@ -1,12 +1,16 @@
-package com.alibaba.dubbo.integration.swagger;
+package org.apache.dubbo.integration.swagger;
```

Comment:
This source file need LICENSE. You could run checkstyle and rat plugin local first, and find out all the problem files.

Reply:
I have fixed these problems. #35